truss/getstarted/template.go

```diff
@@ -0,0 +1,48 @@
+package getstarted
+
+const starterProto = `
+syntax = "proto3";
+
+package {{.PackageName}};
+ 
+import "github.com/TuneLab/go-truss/deftree/googlethirdparty/annotations.proto";
+
+service {{.ServiceName}} {
+  rpc Status(StatusRequest) returns (StatusResponse) {
+    option (google.api.http) = {
+      get: "/Status"
```

Comment:
It was that way originally because it was copy-pasted from a multiverse interface definition, but having it be lowercase is an improvement. Will make the change.